src/main/webapp/index-script.js

```diff
@@ -36,9 +37,20 @@ function loadMainButtons() {
   });
 
   logInButton.addEventListener("click", () => {
-    window.location.href = "/login"
-  });
+    window.location.href="/login" 
+  }); 
+}
+
+function login(){
 
+  const logInButton = document.querySelector("#login");
+  fetch('/login').then(response => response.text()).then(data => {
```

Comment:
do we need to fetch here AND redirect to login on line 40? Can we just do one or the other?

Reply:
Yes - the redirection on line 40 is so users are redirected to the login/logoff status when the button is clicked. This fetch statement is just to see whether a user is logged in when the page loads it can display either login OR logout.